transactions/idTableStaking/admin/add_approved_and_limits.cdc

```diff
@@ -20,40 +20,30 @@ transaction(ids: [String]) {
     }
 
     execute {
-		let nodeIDs = FlowIDTableStaking.getApprovedList()
-			?? panic("Could not read approve list from storage")
-
-		let candidateNodeLimits = FlowIDTableStaking.getCandidateNodeLimits()
-			?? panic("Could not load candidate node limits")
+		let existingApprovedIDs = FlowIDTableStaking.getApprovedList()
+			?? panic("Could not load approved list")
 
 		let slotLimits = FlowIDTableStaking.getRoleSlotLimits()
 
 		// add any new node ID which doesn't already exist in the approve list
 		// and increase the candidate node limits and slot limits by 1
 		// for each corresponding node added
-		for newNodeID in ids {
-			if nodeIDs[newNodeID] != nil {
+		for newNodeID in newApprovedIDs {
+			if existingApprovedIDs[newNodeID] != nil {
     			continue
 			}
 
 			let nodeInfo = FlowIDTableStaking.NodeInfo(newNodeID)
 
-			candidateNodeLimits[nodeInfo.role] = candidateNodeLimits[nodeInfo.role]! + 1
```

Comment:
Why are we not updating candidate limits anymore? Are we going to keep these at high values? (`candidateLimit >> nodeRoleLimit`)

Reply:
Yeah, we already have them at high enough values that we don't need this increase, and when we get to adding in the pre-condition, we'll update them again so that the conditions you recommended are satisfied.